### DISCUSSION TITLE
[feature request] include cname when emitting call events

### DISCUSSION THREAD

**Discussion by mrkiko** [baresip/baresip#3213] [Ideas]
Hello,

I would need an easier (less error-pron) way to obtain the canonical name as visible in /callstats.
I am in the context of implementing a gateway software, thus I need access to this insofmration (see issue #3210).
Please do not convert this issue in a gitHub discussion as well, since this can make it more difficult for me to access your replies.

Does this make sense? Or is there another interface I can leverage?

Thanks a lot again for your great work.

**Comment by sreimers** [comment]
> Please do not convert this issue in a gitHub discussion as well, since this can make it more difficult for me to access your replies.

Issues are only for tracking bugs, we use discussions for feature requests and questions. I see no difference following a issue or a discussion thread here: https://github.com/baresip/baresip/discussions, Notifications and Comment handling are the same. But it helps us to keep the issue list small and clean.
